sensor_msgs/Image rgb_image
---
sensor_msgs/Image depthmap
sensor_msgs/Image residual